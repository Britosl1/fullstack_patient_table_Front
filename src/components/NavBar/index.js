import React from 'react';
import { Navbar, Container } from 'react-bootstrap'

function NavBar() {
  return (
    <Navbar bg="light">
      <Container>
        <Navbar.Brand>Fullstack Form Project</Navbar.Brand>
      </Container>
    </Navbar>
  )
}

export default NavBar;