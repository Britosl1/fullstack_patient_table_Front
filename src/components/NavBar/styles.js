import styled from 'styled-components';

export const NavBarcontainer = styled.nav`
  height: 70px;
  background-color: ${props => props.color};
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0px 20px;
`

export const NavBarText = styled.h1`
  color: ${props => props.color};
  font-size: 30px;
  font-weight: 700;
`

export const NavBarImage = styled.img`
  height: 50px;
  border-radius: 50px;
`