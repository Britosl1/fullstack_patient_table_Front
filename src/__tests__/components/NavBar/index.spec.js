import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../../../components/NavBar';

describe('Snapshot test', () => {
  it('renders correctly', () => {
    const nav = renderer
      .create(<NavBar />)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });

})