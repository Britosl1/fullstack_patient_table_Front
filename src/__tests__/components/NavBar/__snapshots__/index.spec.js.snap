// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders correctly 1`] = `
<nav
  className="sc-bdvvtL dcNLmR"
  color="#E76F51"
>
  <h1
    className="sc-gsDKAQ cRtoSf"
    color="#F4A261"
  >
    Fullstack Form Project
  </h1>
  <img
    className="sc-dkPtRN bVjsqg"
    src="profile.png"
  />
</nav>
`;

exports[`renders correctly 1`] = `
<nav
  className="sc-bdvvtL dcNLmR"
  color="#E76F51"
>
  <h1
    className="sc-gsDKAQ cRtoSf"
    color="#F4A261"
  >
    Fullstack Form Project
  </h1>
  <img
    className="sc-dkPtRN bVjsqg"
    src="profile.png"
  />
</nav>
`;
